control/frdata.py

```diff
@@ -342,110 +335,85 @@ def __pow__(self, other):
             return (FRD(ones(self.fresp.shape), self.omega) / self) * \
                 (self**(other+1))
 
-    def evalfr(self, omega):
-        """Evaluate a transfer function at a single angular frequency.
-
-        self._evalfr(omega) returns the value of the frequency response
-        at frequency omega.
-
-        Note that a "normal" FRD only returns values for which there is an
-        entry in the omega vector. An interpolating FRD can return
-        intermediate values.
-
-        """
-        warn("FRD.evalfr(omega) will be deprecated in a future release "
-             "of python-control; use sys.eval(omega) instead",
-             PendingDeprecationWarning)         # pragma: no coverage
-        return self._evalfr(omega)
-
     # Define the `eval` function to evaluate an FRD at a given (real)
     # frequency.  Note that we choose to use `eval` instead of `evalfr` to
     # avoid confusion with :func:`evalfr`, which takes a complex number as its
     # argument.  Similarly, we don't use `__call__` to avoid confusion between
     # G(s) for a transfer function and G(omega) for an FRD object.
-    def eval(self, omega):
+    # update Sawyer B. Fuller 2020.08.14: __call__ added to provide a uniform 
+    # interface to systems in general and the lti.frequency_response method
+    def eval(self, omega, squeeze=True):
         """Evaluate a transfer function at a single angular frequency.
 
         self.evalfr(omega) returns the value of the frequency response
         at frequency omega.
 
         Note that a "normal" FRD only returns values for which there is an
         entry in the omega vector. An interpolating FRD can return
-        intermediate values.
+        intermediate values.     
+        
+        squeeze: bool, optional (default=True)
+            If True and sys is single input, single output (SISO), return a 
+            1D array or scalar the same length as omega.
 
```

Comment:
```suggestion
        Parameters
        ----------
        squeeze: bool, optional (default=True)
            If True and sys is single input, single output (SISO), return a 
            1D array or scalar the same length as omega.
```

Reply:
Also it misses the entry for `omega`. Summary says single frequency, but the new code also accepts array_like, doesn't it?

Reply:
true. next commit will fix, thanks for catching.